internal/cli/release_create.go

```diff
@@ -156,6 +157,49 @@ func (c *ReleaseCreateCommand) Run(args []string) int {
 				terminal.WithWarningStyle())
 		}
 
+		// Show input variable values used in build
+		// We do this here so that if the list is long, it doesn't
+		// push the deploy/release URLs off the top of the terminal.
+		app.UI.Output("Variables used:", terminal.WithHeaderStyle())
+		headers := []string{
+			"Variable", "Value", "Type", "Source",
+		}
+
+		tbl := terminal.NewTable(headers...)
+		// sort alphabetically for joy
+		inputVars := make([]string, 0, len(result.Release.VariableRefs))
+		for iv := range result.Release.VariableRefs {
+			inputVars = append(inputVars, iv)
+		}
+		sort.Strings(inputVars)
+		for _, iv := range inputVars {
+			// We add a line break in the value here because the Table word wrap
+			// alone can't accomodate the column headers to a long value
+			val := result.Release.VariableRefs[iv].Value
+			if len(val) > 45 {
+				for i := range val {
+					// line break every 45 characters
+					if i%46 == 0 && i != 0 {
+						val = val[:i] + "\n" + val[i:]
```

Comment:
```suggestion
						val = val[:i-1] + "\n" + val[i:]
```

+1 on Brian's comment, would be great to extract this into a helper.